### PR TITLE
Merge with smz3 sprite

### PR DESCRIPTION
**Pull request by Serthow**
Due to Weirdness with github, creadted an extra branch to clone unstable and open a pull request from

### CODE REVIEW

File: alttpo/GameState.as [hunk 1]
```diff
@@ -1112,9 +1137,256 @@ class GameState {
...
+        case 0x7c: return 4;
+        case 0x7d: return 4;
+        case 0x7e: return 4;
+        case 0x7f: return 4;
+        case 0x80: return 4;
+        case 0x81: return 5;
+        case 0x82: return 5;
+        case 0xa4: return 7;
+        case 0xa5: return 7;
+        default: return 0;
```
Comment (JamesDunne) [inline alttpo/GameState.as:1344]: Are you missing 0x19 and 0x1a poses for spin jump here?

Comment (Serthow) [reply alttpo/GameState.as:1344]: for whatever reason those poses appear fine with the default protocol, so i didn't need to list them

Comment (JamesDunne) [reply alttpo/GameState.as:1344]: oh really? i'm guessing it depends on what equipment you have. i know i saw 19 and 1a poses on a clean startup without screw attack

File: alttpo/GameState.as [hunk 2]
```diff
@@ -1112,9 +1137,256 @@ class GameState {
...
+    uint16 address2 = bus::read_u16(0x920000 + offsm2);
+    uint16 size2 = bus::read_u16(0x920000 + offsm2 + 3);
+    uint16 size3 = bus::read_u16(0x920000 + offsm2 + 5);
+    
+    
+    uint32 transfer0;
+    uint32 transfer1;
+    uint16 len1;
+    uint16 len2;
+    if (sm_pose == 0x1a || sm_pose == 0x19 || sm_pose == 0x81 || sm_pose == 0x82 || sm_pose == 0x1b || sm_pose == 0x1c){
```
Comment (Serthow) [inline alttpo/GameState.as:1194]: line 1194: added the space jump poses to the check

**Pull request merged**